Project4/index.html

```diff
@@ -0,0 +1,257 @@
+                    tricks
+                    and
+                    techniques that will help to catch up. Join us and learn short tips that will help you be a
+                    better
+                    After
+                    Effects artist.</p>
+                <p>Speaker: Eran Stern</p>
+                <!--https://nab19.mapyourshow.com/8_0/sessions/session-details.cfm?scheduleid=248 -->
+                <h2>May 14</h2>
+                <p> 3:00 PM - 4:15 PM</p>
+                <p>Creating a TV Channel Ident in After Effects</p>
+                <p>Create a short channel interlude that appears after a commercial break or closing credits.
+                    Animate a
+                    multi-layered illustrator file, use the graph editor with 3D layers, Camera and Lights. Create a
+                    3D
+                    Extrusion, 3D Particles and apply animation presets. Send to client review and finally Render
+                    your
+                    result to
+                    a master file as well as social delivery.
+                    Bring your own laptop and work with the project files in class or just take notes for your next
+                    project.
+                </p>
+                <p>Speaker: Hayley Akins</p>
+                <!-- https://nab19.mapyourshow.com/8_0/sessions/session-details.cfm?scheduleid=190 -->
+                <h2>May 15</h2>
+                <p>10:00 AM - 11:00 AM</p>
+                <p>VLC 4.0</p>
+                <p>A presentation about the next version of VLC, called 4.0.
+                    This is a continuation of the talk from last year, to explain exactly what was indeed
+                    implemented in
+                    the
+                    VLC
+                    core codebase.
+                    A presentation about the next version of VLC, called 4.0, codename Otto Chriek.
+                    This is a continuation of the talk from last year, to explain exactly what was indeed
+                    implemented in
+                    the
+                    VLC
+                    core codebase.
+                    It notably speak about the results of the VLC 3.0 release, and speak about the following changes
+                    for
+                    4.0: -
+                    the media library, - the core clock changes, - the new playlist, - the VR support, - and the new
+                    interface
+                    for Linux and Windows, using Qt5/qml.</p>
+                <p>Speaker: Jean-Baptiste Kempf</p>
+                <!-- https://fosdem.org/2019/schedule/event/media_vlc/ -->
+            </div>
+            </div>
+            <h3 id="sponser">Sponsers</h3>
+            <div class="sponsers image">
+                <a href="https://www.google.com/?client=safari&channel=iphone_bm"><img src="images/google.jpg"
+                        alt="photo of google logo"> </a>
```

Comment:
Your `alt` text doesn't have to include "Photo of..." since that is already clear to screenreaders.
```suggestion
                        alt="Google logo"> </a>
```